protocol/app/ante/market_update_test.go

```diff
@@ -491,64 +491,3 @@ func TestValidateMarketUpdateDecorator_AnteHandle(t *testing.T) {
 		})
 	}
 }
-
-// CreateTestTx is a helper function to create a tx given multiple inputs.
-func CreateTestTx(
```

Comment:
Note: moved to the `testutil/tx` package, and updated to take in a timeout height.